jobs/ci-kubernetes-kubemark-high-density-100-gce.sh

```diff
@@ -40,12 +40,12 @@ export KUBEMARK_TESTS="\[Feature:HighDensityPerformance\]"
 export KUBEMARK_TEST_ARGS="--gather-resource-usage=true --garbage-collector-enabled=true"
 export FAIL_ON_GCP_RESOURCE_LEAK="false"
 # Override defaults to be independent from GCE defaults and set kubemark parameters
-export NUM_NODES="3"
+export NUM_NODES="8"
 export MASTER_SIZE="n1-standard-2"
-export NODE_SIZE="n1-standard-4"
+export NODE_SIZE="n1-standard-8"
 export KUBE_GCE_ZONE="us-east1-d"
 export KUBEMARK_MASTER_SIZE="n1-standard-4"
-export KUBEMARK_NUM_NODES="100"
+export KUBEMARK_NUM_NODES="600"
 # The kubemark scripts build a Docker image
 export JENKINS_ENABLE_DOCKER_IN_DOCKER="y"
 export KUBE_NODE_OS_DISTRIBUTION="gci"
```

Comment:
It would be great if it can be made clearer, like having a variable for it or sth.